### PR TITLE
Add User-Agent for errors logs

### PR DESCRIPTION
**Pull request by VincentCATILLON**
## Detailed purpose of the PR

This PR adds build data to `User-Agent` and defines it in errors logs posted to Datadog HTTP endpoint to determine build version, etc.

##### Through mooc

Capture d’écran 2020-04-29 à 13 15 30

##### Through Datadog endpoint (errors)

Capture d’écran 2020-04-29 à 13 15 45

### Testing strategy

- [x] I ensured my code is tested: unit and manual
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -39,6 +39,8 @@ const styles = StyleSheet.create({
   }
 });
 
+const ENABLE_ERROR_DEBUG = false;
```
Comment (VincentCATILLON) [inline src/app.js:42]: This thing lets you enable error handling in dev mode

File: src/modules/datadog.js [hunk 2]
```diff
@@ -29,68 +18,28 @@ export type LogData = {|
     code?: string
   },
   level: string,
-  build: {
-    environment: BuildEnvironment,
-    type?: BuildType,
-    flavor?: BuildFlavor
-  },
+  ua: string,
```
Comment (VincentCATILLON) [inline src/modules/datadog.js:21]: user agent added to http fetchs

File: src/modules/fetch.js [hunk 3]
```diff
@@ -1,29 +1,15 @@
...
 import get from 'lodash/fp/get';
 
 import {ForbiddenError} from '../models/error';
-import version from './version';
 import {__E2E__} from './environment';
+import {getUserAgent} from './build';
 
 export const ERROR_MESSAGE = 'PLATFORM_DISABLED';
 
-const getUserAgent = async () => {
```
Comment (VincentCATILLON) [inline src/modules/fetch.js:14]: function moved to `src/modules/build`

Comment (Fausto95) [reply src/modules/fetch.js:14]: I don't like the name of the file tho

**Pull request merged**